src/electron/process_manager.ts

```diff
@@ -404,13 +396,33 @@ class Tun2socks extends ChildProcessHelper {
     args.push('--netif-ipaddr', TUN2SOCKS_VIRTUAL_ROUTER_IP);
     args.push('--netif-netmask', TUN2SOCKS_VIRTUAL_ROUTER_NETMASK);
     args.push('--socks-server-addr', `${this.proxyAddress}:${this.proxyPort}`);
-    args.push('--loglevel', this.isInDebugMode ? 'debug' : 'error');
     args.push('--transparent-dns');
     if (isUdpEnabled) {
       args.push('--socks5-udp');
       args.push('--udp-relay-addr', `${this.proxyAddress}:${this.proxyPort}`);
     }
+    args.push('--loglevel', this.isInDebugMode ? 'debug' : 'error');
```

Comment:
tun2socks is quite verbose too, but I don't really see anything useful in the debug logs.
Let's just set the level to `info` and remove the part below.

Reply:
Done in acee18b9a7e13e0554c4e5301128300ba8734d8e